modules/@apostrophecms/schema/ui/apos/mixins/AposInputFollowingMixin.js

```diff
@@ -14,7 +14,7 @@ export default {
         parentFollowing[`<${key}`] = val;
       }
 
-      for (const field of this.field.schema) {
+      for (const field of this.schema) {
```

Comment:
Have we confirmed this mixin is only used in contexts where the schema computed property exists?

Reply:
... I confirmed that. Looks good.